packages/expo-file-system/src/__tests__/FileSystem-test.ts

```diff
@@ -0,0 +1,68 @@
+import { mockPlatformWeb, mockProperty } from 'jest-expo';
+
+import ExponentFileSystem from '../ExponentFileSystem';
+import * as FileSystem from '../FileSystem';
+
+describe('FileSystem', () => {
+  describe('Constants', () => {
+    it('documentDirectory', () => expect(FileSystem.documentDirectory).toBeDefined());
+    it('cacheDirectory', () => expect(FileSystem.cacheDirectory).toBeDefined());
+    it('bundledAssets', () => expect(FileSystem.bundledAssets).toBeDefined());
+    it('bundleDirectory', () => expect(FileSystem.bundleDirectory).toBeDefined());
+  });
+
+  describe('Methods', () => {
+    const URI = '/';
+    const toURI = '/other';
+    it('downloadAsync', () => {
+      const props: any = ['foo', 'bar', {}];
+      FileSystem.downloadAsync(props[0], props[1], props[2]);
+      expect(ExponentFileSystem.downloadAsync).toHaveBeenCalledWith(...props);
+    });
+
+    it('getInfoAsync', async () => {
+      await FileSystem.getInfoAsync(URI);
```

Comment:
For these tests, it'd be much better to pick some tricky functionality in the API layer that is either likely to break with a future code change (unintentionally of course) or is nuanced behavior that you want to demonstrate with code.

Specifically for `getInfoAsync`, the API layer doesn't do much and the code is so simple that we can verify what it's supposed to do without a test -> I'd just remove this.

OTOH, the DownloadResumable class has more complexity in the API layer and unit tests for it would be more helpful.